pages/home.jsx

```diff
@@ -1,19 +1,157 @@
 var React = require('react');
 var Router = require('react-router');
 var Link = Router.Link;
+var moment = require('moment');
 
 var HeroUnit = require('../components/hero-unit.jsx');
 var Blockquote = require('../components/blockquote.jsx');
 var Illustration = require('../components/illustration.jsx');
+var IconLinks = require('../components/icon-links.jsx');
+var IconLink = require('../components/icon-link.jsx');
+
+var config = require('../lib/config');
 
 var CaseStudies = React.createClass({
   render: function() {
     return (
-      <div className="case-studies">
-        <Blockquote className="primary-quote" author="Maurya C. New York, United States"
-            imgSrc="/img/pages/home/maurya-nyc.png" imgSrc2x="/img/pages/home/maurya-nyc@2x.png" imgAlt="Maurya NYC Quote">
-          <p>Web literacy is about more than coding - it's about how you can be a better web citizen.</p>
-        </Blockquote>
+      <div className="row">
+        <div className="col-sm-offset-2 col-sm-8 col-md-offset-2 col-md-8 col-lg-offset-2 col-lg-8">
+          <div>
+            <Blockquote author="Maurya C. New York, United States"
+                imgSrc="/img/pages/home/maurya-nyc.png" imgSrc2x="/img/pages/home/maurya-nyc@2x.png">
+              <p>"Web literacy is about more than coding - it's about how you can be a better web citizen."</p>
+            </Blockquote>
+          </div>
+        </div>
+      </div>
+    );
+  }
+});
+
+var FeaturedPost = React.createClass({
+  propTypes: {
+    data: React.PropTypes.object.isRequired
+  },
+  render: function() {
+    var parsedDate = moment(this.props.data.publishedDate);
+    return(
+      <div className="featured-post">
+        <div className="entry-posted-container">
+          <p className="entry-posted">
+            <time className="published" title={this.props.data.publishedDate} dateTime={this.props.data.publishedDate} >
```

Comment:
leaving `dateTime` value unparsed here since it's in machine readable format already

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time
> This element is intended to be used presenting dates and times in a machine readable format. This can be helpful for user agents to offer any event scheduling for user's calendar.